packages/mask/src/extension/popups/hooks/useFriends.ts

```diff
@@ -93,5 +98,32 @@ export function useFriendsPaged() {
         refetch,
         status,
         fetchRelationStatus,
+        records,
     }
 }
+
+export function useFriendFromList(searchedRecords: RelationRecord[]) {
+    const currentPersona = useCurrentPersona()
+    return useQuery(['search-local', searchedRecords], async () => {
+        const friends: Friend[] = (
+            await Promise.all(
+                searchedRecords.map(async (x) => {
+                    if (isProfileIdentifier(x.profile)) {
+                        const res = first(await Services.Identity.queryProfilesInformation([x.profile]))
+                        if (res?.linkedPersona !== undefined && res?.linkedPersona !== currentPersona?.identifier)
```

Comment:
These two object might not equal, comparing by public key instead.
```suggestion
                        if (res?.linkedPersona !== undefined && res?.linkedPersona.publicKeyAsHex !== currentPersona?.identifier.publicKeyAsHex)
```